app/models/art_item.rb

```diff
@@ -20,8 +20,16 @@ class ArtItem < ApplicationRecord
   belongs_to :department
   has_many :annotations
   has_rich_text :description
+  has_one :description, class_name: 'ActionText::RichText', as: :record
```

Comment:
What is this doing?

Reply:
Because if you want to look at the application on your localhost you need to add these records to your database

Reply:
You need this to run ransack search through ActionText fields (look at the VOD code)